### PR TITLE
Update the guide to the new format

### PR DESCRIPTION
**Pull request by mtyazici**

### CODE REVIEW

File: doc/modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -0,0 +1,149 @@
+////
+This is the base template for Hazelcast integration module guides.
+
+You can clone this repository, work on it and create your guide, and then push to a new repository.
+////
+
+:github-address: [github-repo-address-of-your-guide]
```
Comment (mtyazici) [inline doc/modules/ROOT/pages/index.adoc]:
```suggestion
:github-address: https://github.com/hazelcast-guides/springboot-tomcat-session-replication
```

File: doc/antora.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+name: springboot-tomcat-session-replication    # a brief module name. This will be included in the guide url.
+title:  Spring Boot Tomcat Session Replication using Hazelcast # a brief module title. This will not show up on the website.
+version: master
+
```
Comment (enozcan) [inline doc/antora.yml]:
```suggestion
name: springboot-tomcat-session-replication
title:  Spring Boot Tomcat Session Replication using Hazelcast
version: master
```

File: doc/modules/ROOT/pages/index.adoc [hunk 3]
```diff
@@ -0,0 +1,149 @@
+////
+This is the base template for Hazelcast integration module guides.
+
+You can clone this repository, work on it and create your guide, and then push to a new repository.
+////
+
```
Comment (enozcan) [inline doc/modules/ROOT/pages/index.adoc]:
```suggestion
```

File: doc/modules/ROOT/pages/index.adoc [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+:github-address: https://github.com/hazelcast-guides/springboot-tomcat-session-replication
+:hazelcast: Hazelcast IMDG
+:framework: Spring Boot
+
+// Use this relative url if you are going to publish the guide on the guides site.
+// Note that this url will not work locally and raise asciidoctor errors.
+// So, complete the guide with the above url and set the below one just before 
+// publishing on the guides site.
+//
```
Comment (enozcan) [inline doc/modules/ROOT/pages/index.adoc]:
```suggestion
```

File: doc/modules/ROOT/pages/index.adoc [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+:templates-url: templates:ROOT:page$
+
+= Spring Boot Tomcat Session Replication using Hazelcast
+
+// Content entered directly below the header but before the first section heading is called the preamble.
```
Comment (enozcan) [inline doc/modules/ROOT/pages/index.adoc]: Please remove the comments from the base guide next time.
```suggestion
```

File: doc/modules/ROOT/pages/index.adoc [hunk 6]
```diff
@@ -0,0 +1,149 @@
...
+- The first bean creates a Hazelcast `Config` object to configure Hazelcast members. We enable the TCI/IP configuration.
+- The second bean creates the Hazelcast member using the `hazelcastConfig` bean.
+- The third one customizes Tomcat instance in Spring Boot to use Hazelcast Tomcat Session Manager.
+
+Please note that the Hazelcast instance name for `HazelcastSessionManager` object and the instance name for Hazelcast config bean should be the same. Otherwise, `HazelcastSessionManager` wouldn't access the running Hazelcast instance.
+
+== Running the Sample Application
+
+
+Build the application by using maven, make sure you are in the project directory
```
Comment (enozcan) [inline doc/modules/ROOT/pages/index.adoc]: Let's simplify this
```suggestion
In the project directory, build the application:
```

File: doc/modules/ROOT/pages/index.adoc [hunk 7]
```diff
@@ -0,0 +1,149 @@
...
+[source,bash]
+----
+mvn clean package
```
Comment (enozcan) [inline doc/modules/ROOT/pages/index.adoc]: It's obviously a shell command, but we've done in the other guides this way
```suggestion
$ mvn clean package
```

File: doc/modules/ROOT/pages/index.adoc [hunk 8]
```diff
@@ -0,0 +1,149 @@
...
+----
+
+Run the application by typing
```
Comment (enozcan) [inline doc/modules/ROOT/pages/index.adoc]: Seems like no other option than typing :)
```suggestion
Run the application:
```

File: doc/modules/ROOT/pages/index.adoc [hunk 9]
```diff
@@ -0,0 +1,149 @@
...
+[source,bash]
+----
+java -Dserver.port=8080 -jar target/springboot-tomcat-session-replication-0.1.0.jar
```
Comment (enozcan) [inline doc/modules/ROOT/pages/index.adoc]: See above
```suggestion
$ java -Dserver.port=8080 -jar target/springboot-tomcat-session-replication-0.1.0.jar 
```

File: doc/modules/ROOT/pages/index.adoc [hunk 10]
```diff
@@ -0,0 +1,149 @@
...
+
+Run the application by typing
+[source,bash]
+----
+java -Dserver.port=8080 -jar target/springboot-tomcat-session-replication-0.1.0.jar 
+----
+
+
+
+Open another terminal and rerun the application by typing
```
Comment (enozcan) [inline doc/modules/ROOT/pages/index.adoc]:
```suggestion
Open another terminal and start the second application:
```

File: doc/modules/ROOT/pages/index.adoc [hunk 11]
```diff
@@ -0,0 +1,149 @@
...
+For each command the output should be like the following:
+----
+{"value":"hazelcast"}
+----
+
+Notice we have to use cookies when testing the application since the data is saved in HTTP sessions.
+
+== Summary
+
+We created a Spring Boot application that accesses session scope data. To be able to share the session data between each application we used Hazelcast integration. When we  put the key-value pair to our current session as an attribute at port 8080 and we could get the attributes from another port. So we accomplished keeping the session data in each microservice application.
```
Comment (enozcan) [inline doc/modules/ROOT/pages/index.adoc]:
```suggestion
We created a Spring Boot application that accesses session scope data. To be able to share the session data between each application we used Hazelcast integration. When we  put the key-value pair to our current session as an attribute at port 8080, we could get the attributes from another port. So we accomplished keeping the session data in each microservice application.
```

File: doc/modules/ROOT/pages/index.adoc [hunk 12]
```diff
@@ -0,0 +1,149 @@
...
+
+== See Also
+
+// Add some links to resources, such as other related guides.
+// Use relative links used on the home page (see https://raw.githubusercontent.com/hazelcast-guides/guides-site/master/home/modules/ROOT/pages/index.adoc)
+
+- xref:hazelcast-embedded-springboot:ROOT:index.adoc[Hazelcast in SpringBoot]
+- xref:hazelcast-quarkus:ROOT:index.adoc[Hazelcast Client for Quarkus]
```
Comment (enozcan) [inline doc/modules/ROOT/pages/index.adoc]:
```suggestion
- xref:hazelcast-embedded-springboot:ROOT:index.adoc[Getting Started with Hazelcast using Spring Boot]
- xref:springboot-session-replication:ROOT:index.adoc[Filter Based Session Replication with Spring Boot and Hazelcast] 
```

File: src/main/java/guides/hazelcast/tomcatsessionmanager/Application.java [hunk 13]
```diff
@@ -0,0 +1,60 @@
+package guides.hazelcast.tomcatsessionmanager;
+
+import com.hazelcast.config.Config;
+import com.hazelcast.config.JoinConfig;
+import com.hazelcast.core.Hazelcast;
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.session.HazelcastSessionManager;
+import org.springframework.beans.factory.annotation.Value;
```
Comment (enozcan) [inline src/main/java/guides/hazelcast/tomcatsessionmanager/Application.java]: This import seems unused. Can you please also check other ones if they are necessary?

File: src/main/java/guides/hazelcast/tomcatsessionmanager/CommandController.java [hunk 14]
```diff
@@ -0,0 +1,29 @@
+package guides.hazelcast.tomcatsessionmanager;
+
+import org.springframework.beans.factory.annotation.Value;
```
Comment (enozcan) [inline src/main/java/guides/hazelcast/tomcatsessionmanager/CommandController.java]: See previous review

File: src/main/java/guides/hazelcast/tomcatsessionmanager/CommandController.java [hunk 15]
```diff
@@ -0,0 +1,29 @@
...
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.servlet.http.HttpSession;
+import java.util.Map;
```
Comment (enozcan) [inline src/main/java/guides/hazelcast/tomcatsessionmanager/CommandController.java]: I do not see any maps in use

File: src/main/java/guides/hazelcast/tomcatsessionmanager/CommandResponse.java [hunk 16]
```diff
@@ -0,0 +1,25 @@
+package guides.hazelcast.tomcatsessionmanager;
+
+@SuppressWarnings("unused")
```
Comment (enozcan) [inline src/main/java/guides/hazelcast/tomcatsessionmanager/CommandResponse.java]: Is there even a complaint regarding this?

**Review by enozcan — CHANGES_REQUESTED** [review]
Comments provided. Also please mind the comments on the base guide abstraction and avoid extra spaces, newlines, etc.

File: doc/modules/ROOT/pages/index.adoc [hunk 17]
```diff
@@ -0,0 +1,149 @@
...
+
+= Spring Boot Tomcat Session Replication using Hazelcast
+
+// Content entered directly below the header but before the first section heading is called the preamble.
+
+include::{templates-url}/link-to-repo.adoc[]
+
+== What You’ll Learn
+
+Spring Boot is a framework that helps to build microservices easily. In a microservice architecture, there are different options to share data among the running services: one of them is caching, which Spring Boot leverages. It also provides session scope data access using built-in Tomcat HTTP sessions. But when running multiple microservices, it quickly becomes a hassle to replicate sessions and share their data across microservices.
```
Comment (enozcan) [inline doc/modules/ROOT/pages/index.adoc]: I wonder why did you trim the [original content](https://github.com/hazelcast-guides/springboot-tomcat-session-replication-on-kubernetes#spring-boot-tomcat-session-replication-on-kubernetes-using-hazelcast) here.

File: src/main/java/guides/hazelcast/tomcatsessionmanager/Application.java (hunk 13 above)
Comment (mtyazici) [reply src/main/java/guides/hazelcast/tomcatsessionmanager/Application.java]: I will have to check them

File: doc/modules/ROOT/pages/index.adoc (hunk 17 above)
Comment (mtyazici) [reply doc/modules/ROOT/pages/index.adoc]: I can put the original content back. I wasn't sure if that would be too long, I actually liked it and had troubles trimming it.

Comment (enozcan) [reply doc/modules/ROOT/pages/index.adoc]: Let's bring it back. We can divide into multiple categories afterwards.

Comment (mtyazici) [reply doc/modules/ROOT/pages/index.adoc]: Alright I will do it

**Review by enozcan — APPROVED** [review]
Thanks!

**Pull request merged**